dvc/command/status.py

```diff
@@ -68,6 +69,14 @@ def run(self):
                 self._show(st, indent)
             elif not self.repo.stages:
                 logger.info(self.EMPTY_PROJECT_MSG)
+            elif self.args.cloud:
+                logger.info(
+                    self.IN_SYNC_MSG.format(
+                        remote=self.repo.config["core"].get("remote")
+                    )
+                )
+            elif self.args.remote:
+                logger.info(self.IN_SYNC_MSG.format(remote=self.args.remote))
```

Comment:
@tenitski Thanks! Looks good! One minor thing: let's keep one branch of execution:
```
            elif self.args.cloud:
                remote = self.args.remote or self.repo.config["core"].get("remote")
                logger.info(self.IN_SYNC_MSG.format(remote=remote))
```
just a bit easier to comprehend that way :slightly_smiling_face:

Reply:
Done. Linter insisted on breaking it into a few more lines though

Reply:
Yeah, that's expected, linter sometimes makes it a little ugly :smile: Could refactor to fit into 80 chars or could leave as is, I'm fine with either :ok_hand:

Reply:
Makes sense, I didn't realise line length was the trigger.